src/OpenTelemetry.Instrumentation.ConfluentKafka/AssemblyInfo.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("OpenTelemetry.Instrumentation.ConfluentKafka.Tests" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2" + AssemblyInfo.MoqPublicKey)]
```

Comment:
Is this being used? I don't see Moq being used anywhere in the PR (but I might be missing something).

Reply:
dropped. Moq has been removed since I opened the PR, i adapted the unit tests but forgot this. thanks